examples/ecs/nodejs/index.ts

```diff
@@ -29,6 +29,8 @@ const service = new ecs.FargateService("my-service", {
         subnets: vpc.publicSubnetIds,
         assignPublicIp: true,
     },
+}, {
+    dependsOn: [lb.loadBalancer],
```

Comment:
Should we add logic to the `FargateService` component to automatically add the dependency on the load balancer?

Reply:
Based on our conversation I think we concluded given the component is under the classic portion of the provider our preference would be to sanitize examples instead to not rely on those components.